### PR TITLE
ESLint updates

### PR DESCRIPTION
**Pull request by rolyatmax**
Separating this out from https://github.com/onefinestay/react-daterange-picker/pull/78

### CODE REVIEW

**Comment by rolyatmax** [comment]
@AlanFoster Split this out into its own PR. Let me know what you think!

Also, do you think this should this be split into two different PRs? Or is one fine?

**Comment by AlanFoster** [comment]
@rolyatmax Brilliant, thanks! Let's split this into one more PR, that'd be great :+1:

**Comment by rolyatmax** [comment]
Split out build updates to #81

**Comment by rolyatmax** [comment]
@AlanFoster Take a look and let me know what you think!

File: package.json [hunk 1]
```diff
@@ -36,9 +36,12 @@
   "devDependencies": {
     "babel": "^5.2.16",
     "babel-core": "^5.2.6",
+    "babel-eslint": "^4.1.3",
     "babel-loader": "^5.0.0",
     "babel-plugin-object-assign": "^1.0.1",
     "brfs": "^1.2.0",
+    "eslint": "^1.7.3",
```
Comment (AlanFoster) [inline package.json]: :question: Do we need to install these locally if there are no gulp tasks that make use of the dependencies?

Comment (rolyatmax) [inline package.json]: This is useful as it doesn't require someone to install `eslint` globally to have linting work. It seems to be as much of one of the `devDependencies` any of the others in the list. Perhaps we should add linting to the npm scripts?

```json
"scripts": {
  "lint": "eslint"
}
```

and then:

```bash
npm run lint
```

Comment (AlanFoster) [inline package.json]: :+1: We should definitely add an entry point for these rules within our `package.json` file; We can then extend this to use a custom Gulp task in the future if needed. Happy to merge after that change has been added.

Comment (rolyatmax) [inline package.json]: A lot of devs will have their editor set up to do linting automatically. Mine, for example, sees an `.eslintrc` in the root and then looks for a local `eslint` binary. This makes it easier for me to switch between projects that use `eslint` or `jshint`.

Comment (AlanFoster) [inline package.json]: :+1: Happy to merge this. Thanks!

**Pull request merged**